data/test-cases/plugins/ms-teams-sync/UI-design-1.0/RHS_design.md

```diff
@@ -0,0 +1,46 @@
+---
+# (Required) Ensure all values are filled up
+name: "Validating the RHS design"
```

Comment:
Is the name correct here, as we are not checking the whole rhs functionality but instead just enabling and disabling the app bar?